src/Altinn.Notifications.Core/Models/NotificationTemplate/SmsTemplate.cs

```diff
@@ -0,0 +1,39 @@
+using Altinn.Notifications.Core.Enums;
+
+namespace Altinn.Notifications.Core.Models.NotificationTemplate;
+
+/// <summary>
+/// Template for a SMS notification
+/// </summary>
+public class SmsTemplate : INotificationTemplate
+{
+    /// <inheritdoc/>
+    public NotificationTemplateType Type { get; internal set; }
+
+    /// <summary>
+    /// Gets the number from which the SMS is created by the template    
+    /// </summary>
+    public string SenderNumber { get; internal set; } = string.Empty;
```

Comment:
Same might apply here? Do we want to use the word number or could we use something else ?

Reply:
Could `SenderHandle` be an option? 🤔

Reply:
Going with `SenderHandle` as Terje approved in the daily standup. So marking this as resolved.

Reply:
SenderNumber describes the property better than SenderHandle in my opinion. 
The documentation should describe that this could be number or string.

Reply:
Regardless of what we end up with I think the same term should be used in NotificationOrderConfig and the SMS template as they refer to the same concept

Reply:
- [Renamed SenderHandle to SenderNumber again as agreed](https://github.com/Altinn/altinn-notifications/pull/380/commits/c0c20dba91e2f1de7bf1729dc4b99b63202b7f13)
- [Renamed NotificationOrderConfig default proporty for SMS as agreed](https://github.com/Altinn/altinn-notifications/pull/380/commits/1cfa0a7f4a977e0a50c3a98def309a3930e92de3)